README.md

```diff
@@ -3,39 +3,42 @@
 
 ## [Learning Bitmark](/pages/learning-bitmark/README.md)
 
-**[Bitmark Property System Introduction](/pages/learning-bitmark/problem-we-are-trying-to-solve.md)**
+**[Property System Introduction](/pages/learning-bitmark/problem-we-are-trying-to-solve.md)**
 
-**[Bitmark Blockchain Introduction](/pages/learning-bitmark/bitmark-blockchain.md)**
+**[Blockchain Introduction](/pages/learning-bitmark/bitmark-blockchain.md)**
 
 **[Quick Start](/pages/learning-bitmark/quick-start/README.md)**
-* [Running a Bitmark Node](/pages/learning-bitmark/quick-start/simple-solution-for-node-setup.md#run-a-bitmark-node)
-* [Creating a Bitmark Account](/pages/learning-bitmark/quick-start/working-with-bitmarks/creating-bitmark-account.md)
-* [Registering Bitmark Certificates](/pages/learning-bitmark/quick-start/working-with-bitmarks/issuing-bitmarks.md)
-* [Transferring Bitmark Certificates](/pages/learning-bitmark/quick-start/working-with-bitmarks/transferring-bitmarks.md)
-* [Using Bitmark Shares](/pages/learning-bitmark/quick-start/working-with-bitmarks/using-bitmark-shares.md)
+* [Running a Bitmark Node](/pages/learning-bitmark/quick-start/simple-solution-for-node-setup.md)
+* [Working with Bitmark](/pages/learning-bitmark/quick-start/working-with-bitmarks/working-with-bitmark.md)
+* [Using the Bitmark app](/pages/learning-bitmark/quick-start/working-with-bitmarks/using-bitmark-app.md)
```

Comment:
It seems a bit inconsistent calling this the "Bitmark app" but having the others just be "the SDK" and "the CLI". Perhaps call it "the mobile app", which maintains specificity but keeps this from being the only one named Bitmark?

Reply:
The app name on AppStore is "Bitmark"

Reply:
I would still suggest "mobile app", since Bitmark is used in a few other ways here, but I could see going the other way.